### ISSUE TITLE
Visualize Octree.

### ISSUE THREAD

**Issue by Haider-BA** [brandonpelfrey/SimpleOctree#4]
How to visualize Octree in Tecplot or Paraview or VisIt? If want to couple with some CFD code, is it possible?

**Comment by brandonpelfrey** [comment]
https://www.youtube.com/watch?v=IoCLhiOPi-I

Are you wanting to do something like the above, where each 'cube' in the octree is drawn separately?

**Comment by Haider-BA** [comment]
Yes, something like the video, and want to implement this method: [Grid refinement in lattice Boltzmann methods based on volumetric formulation](http://www.sciencedirect.com/science/article/pii/S0378437105009696) both in 2D(Quadtree) and 3D (Octree). Is it possible to specialize it for quadtree also? 

[https://www.youtube.com/watch?v=RtSh_a4lx_U](**url**)

![3d](https://user-images.githubusercontent.com/10229251/29248298-9ebbd512-804f-11e7-986e-af8cf5ec598c.jpg)
<img width="168" alt="structured-refined-grid-in2d-2d" src="https://user-images.githubusercontent.com/10229251/29248299-a178af1e-804f-11e7-8542-f47dfe12c6e9.png">
